packages/fiori/src/Timeline.ts

```diff
@@ -105,14 +117,22 @@ class Timeline extends UI5Element {
 	}
 
 	_onfocusin(e: FocusEvent) {
-		const target = e.target as TimelineItem;
+		let target = e.target as ITimelineItem | ToggleButton;
+
+		if ((target as ITimelineItem).isGroupItem) {
+			target = target.shadowRoot!.querySelector<ToggleButton>("ui5-toggle-button")!;
```

Comment:
when we want to query a specific tag we are usign attribute selection due to scoping.